test/ActiveLogin.Authentication.BankId.Api.Test/BankIdApiClientExtensions_Tests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using ActiveLogin.Authentication.BankId.Api.Models;
+using ActiveLogin.Authentication.BankId.Api.Test.TestHelpers;
+using Moq;
+using Xunit;
+// ReSharper disable InvokeAsExtensionMethod
+
+namespace ActiveLogin.Authentication.BankId.Api.Test
+{
+    public class BankIdApiClientExtensions_Tests
+    {
+        [Fact]
+        public async void AuthAsync_WithEndUserIp_ShouldMap_ToAuthRequest_WithEndUserIp()
+        {
+            // Arrange
+            var bankIdApiClientMock = new Mock<IBankIdApiClient>(MockBehavior.Strict);
```

Comment:
Was the strict mode chosen by purpose? :)

Reply:
Hard to say, I started reading about it and found an article with the debates, it is like tabs vs spaces :) I think in that case is better to keep it strict - would be much easier to track failed tests if we add new logic in some methods (like method calling another method - this is the most valuable argument in these debates)